warehouse/forklift/legacy.py

```diff
@@ -1033,7 +1036,8 @@ def file_upload(request):
             raise _exc_with_message(
                 HTTPBadRequest,
                 "This filename has previously been used, you should use a "
-                "different version.",
+                "different version. "
+                "See https://pypi.org/help/#file-name-reuse",
```

Comment:
Could you generate these URLs with `request.route_url('help', _anchor='file-name-reuse')` instead? Otherwise these will be wrong for https://test.pypi.org/

Reply:
Sure, Will address.

Reply:
@di What about `_scheme and  _host` ? Wouldn't `request.route_url('help', _anchor='file-name-reuse')` just generate the path?

Reply:
[`route_path`](https://docs.pylonsproject.org/projects/pyramid/en/latest/api/request.html#pyramid.request.Request.route_path) would generate a path, like `/help/#file-name-reuse`, [`route_url`](https://docs.pylonsproject.org/projects/pyramid/en/latest/api/request.html#pyramid.request.Request.route_url) generates a fully qualified URL, like  `https://pypi.org/help/#file-name-reuse`, so you don't need to specify a host or scheme.